src/index.html

```diff
@@ -36,6 +40,10 @@ <h2 class="error-title">Error Loading Data</h2>
 
       <!-- Data State -->
       <ul id="network-list" class="network-list hidden">
+        <li class="network-item header">
+          <div class="network-info header-cell">Network</div>
+          <div class="success-rate header-cell">Retrievability Score</div>
+        </li>
```

Comment:
This is not based on the design created by Lumen, correct? What is the goal here? Shouldn't we instead turn the Lumen design into code?

Reply:
@juliangruber Yes it's not based on their design. Idea here was to improve existing leaderboard while waiting for their design -- not knowing that their design was completed at the time.